internal/cmd/stack/environment.go

```diff
@@ -140,16 +150,37 @@ func (e *listEnvCommand) listEnv(cliCtx *cli.Context) error {
 	for _, config := range query.Stack.RuntimeConfig {
 		config := config
 		var contextName *string
+		var isAutoAttached *bool
 		if config.Context != nil {
 			contextName = &config.Context.ContextName
+
+			f := false
+			isAutoAttached = &f
```

Comment:
```
isAutoAttached = &[]bool{f}[0]
```

I saw this "trick" yesterday, I don't yet know if I like it or not.

Reply:
I'm a man of few tricks and this will not be another one of them.